firebase-firestore/src/main/java/com/google/firebase/firestore/local/LocalViewChanges.java

```diff
@@ -70,6 +73,14 @@ public int getTargetId() {
     return targetId;
   }
 
+  /**
+   * Returns whether there were any unresolved limbo documents in the corresponding view when this
+   * change was computed.
+   */
+  public boolean hasUnresolvedLimboDocuments() {
```

Comment:
Note that I picked this name to suggest that the limbo state of the ViewChange doesn't reflect whether the change itself contains limbo documents.